### ISSUE TITLE
Add a `contributing.md`

### ISSUE THREAD

**Issue by arthurvr** [gabrielelana/erlang-watchlist#1]
With clear guidelines when and how to contribute. Also needs to include how the list is ordered and how it's structured.

**Comment by gabrielelana** [comment]
You are right, we miss a contribution guideline, that's because we are really informal, if you care to contribute please do with a pull request, in the meantime we will provide a `CONTRIBUTE.md` :wink:
